06 week/player/app.js

```diff
@@ -0,0 +1,41 @@
+const gameContainer = document.getElementById("gameContainer");
+const player = document.getElementById("player");
+
+let topGameContainer = gameContainer.offsetTop;
+let leftGameContainer = gameContainer.offsetLeft;
+let widthGameContainer = gameContainer.offsetWidth;
+let heightGameContainer = gameContainer.offsetHeight;
+document.addEventListener("keydown", function (e) {
+  if (e.code == "ArrowDown") {
```

Comment:
in loc sa tot pui if-uri dupa fiecare directie poti imparti in doua logica
1. Prima data calculezi nextTop si nextLeft
2. Faci if daca nextTop si nextLeft sunt in container iar daca da, faci update la player in dom